src/core/data.jl

```diff
@@ -250,24 +250,28 @@ function _apply_func!(data::Dict{String,<:Any}, key::String, func)
     end
 end
 
-
 "Transforms network data into per-unit"
 function make_per_unit!(data::Dict{String,<:Any})
+    _IM.modify_data_with_function!(data, _pm_it_name, _make_per_unit!; apply_to_nws = false)
+end
+
+"Transforms network data into per-unit"
+function _make_per_unit!(data::Dict{String,<:Any})
     if !haskey(data, "per_unit") || data["per_unit"] == false
         data["per_unit"] = true
         if _IM.ismultinetwork(data)
             for (i,nw_data) in data["nw"]
-                _make_per_unit!(nw_data)
+                _make_per_unit_nw!(nw_data)
             end
         else
-            _make_per_unit!(data)
+            _make_per_unit_nw!(data)
         end
     end
 end
 
 
 ""
-function _make_per_unit!(data::Dict{String,<:Any})
+function _make_per_unit_nw!(data::Dict{String,<:Any})
```

Comment:
At least for the moment, I think it would be best to keep this function name as `_make_per_unit!`

Reply:
Should be doable after implementing the previous suggestion.